helm-charts/common/llm-uservice/README.md

```diff
@@ -1,55 +1,90 @@
+
+# for llm-docsum service
+curl http://localhost:9000/v1/docsum \
+  -X POST \
+  -d '{"query":"Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5.", "max_tokens":32, "language":"en"}' \
+  -H 'Content-Type: application/json'
+
+# for llm-faqgen service
+curl http://localhost:9000/v1/faqgen \
+  -X POST \
+  -d '{"query":"Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5.","max_tokens": 128}' \
+  -H 'Content-Type: application/json'
 ```
 
 ## Values
 
-| Key                             | Type   | Default          | Description                     |
-| ------------------------------- | ------ | ---------------- | ------------------------------- |
-| global.HUGGINGFACEHUB_API_TOKEN | string | `""`             | Your own Hugging Face API token |
-| image.repository                | string | `"opea/llm-tgi"` |                                 |
-| service.port                    | string | `"9000"`         |                                 |
-| TGI_LLM_ENDPOINT                | string | `""`             | LLM endpoint                    |
-| global.monitoring               | bool   | `false`          | Service usage metrics           |
+| Key                             | Type   | Default                       | Description                                                                      |
+| ------------------------------- | ------ | ----------------------------- | -------------------------------------------------------------------------------- |
+| global.HUGGINGFACEHUB_API_TOKEN | string | `""`                          | Your own Hugging Face API token                                                  |
+| image.repository                | string | `"opea/llm-textgen"`          | one of "opea/llm-textgen", "opea/llm-docsum", "opea/llm-faqgen"                  |
+| LLM_ENDPOINT                    | string | `""`                          | backend inference service endpoint                                               |
+| LLM_MODEL_ID                    | string | `"Intel/neural-chat-7b-v3-3"` | model used by the inference backend                                              |
+| TEXTGEN_BACKEND                 | string | `"tgi"`                       | backend inference engine, only valid for llm-textgen image, one of "TGI", "vLLM" |
+| DOCSUM_BACKEND                  | string | `"tgi"`                       | backend inference engine, only valid for llm-docsum image, one of "TGI", "vLLM"  |
+| FAQGEN_BACKEND                  | string | `"tgi"`                       | backend inference engine, only valid for llm-faqgen image, one of "TGi", "vLLM"  |
```

Comment:
ConfigMap template does case-sensitive comparison, so the default values need to be updated too: `tgi` -> `TGI`.

Reply:
since it's already merged, I'll do that in another PR.